Tools/ardupilotwaf/cxx_checks.py

```diff
@@ -140,10 +140,11 @@ def check_librt(cfg):
         mandatory=False,
     )
 
-    if success:
-        return success
+    if ret:
+        return ret
 
-    return cfg.check(
-        compiler='cxx',
-        lib='rt',
-    )
+    ret = cfg.check(compiler='cxx', lib='rt', mandatory=True)
```

Comment:
`mandatory` is `True` by default. So I'd prefer to remove it. Could you please do so and fix the commit message?